runtime/src/accounts_index.rs

```diff
@@ -1524,7 +1524,7 @@ impl<T: IndexValue> AccountsIndex<T> {
         );
     }
 
-    pub(crate) fn get_account_maps_read_lock(&self, pubkey: &Pubkey) -> AccountMapsReadLock<T> {
+    pub(crate) fn get_bin(&self, pubkey: &Pubkey) -> AccountMapsReadLock<T> {
```

Comment:
`AccountMapsReadLock` is not a ReadLock any more too. 
Maybe rename it to `AccountMapRef` next?